oms-monitor/docker/logstash/pipeline/logstash.conf

```diff
@@ -118,17 +118,9 @@ filter {
 }
 
 output {
-  if "beat" in [tags] {
-    elasticsearch {
-      hosts => "elasticsearch:9200"
-        # manage_template => false
-      index => "%{[@metadata][beat]}-%{[@metadata][version]}-%{+YYYY.MM.dd}"  ###https://www.elastic.co/guide/en/logstash/current/plugins-inputs-beats.html
-    }
-  } else {
-    elasticsearch {
-      hosts => "${ELASTIC_INSTANCE}"
-      user => "${ELASTIC_USER}"
-      password => "${ELASTIC_PASSWORD}"
-    }
+  elasticsearch {
+    hosts => "${ELASTIC_HOST}"
+    user => "${ELASTIC_USER}"
```

Comment:
so this works without even a `envsubst` ?? wow

Reply:
yep: https://www.elastic.co/guide/en/logstash/current/environment-variables.html